Dockerfile

```diff
@@ -3,5 +3,6 @@ MAINTAINER Democracy Works, Inc. <dev@democracy.works>
 
 COPY ./start /bin/start
 
+ADD standalone-rsyslog.xml /opt/jboss/wildfly/standalone/configuration/standalone-rsyslog.xml
+
 ENTRYPOINT ["/bin/start"]
-CMD ["-b", "0.0.0.0", "-bmanagement", "0.0.0.0"]
```

Comment:
We may want a `CMD []` in here now just to ensure we don't inherit anything from upstream.